test/CMakeLists.txt

```diff
@@ -28,15 +28,21 @@ foreach (bustub_test_source ${BUSTUB_TEST_SOURCES})
     add_dependencies(build-tests ${bustub_test_name})
     add_dependencies(check-tests ${bustub_test_name})
 
+    SET(TEST_TIMEOUT 120)
+
+    if(${bustub_test_name} STREQUAL "grading_b_plus_tree_checkpoint_2_concurrent_test")
```

Comment:
I would prefer setting a global timeout of i.e. 300s or removing the timeout. The grading script already have timeout for each test case 🤣